src/libs/OptionsListUtils.ts

```diff
@@ -1119,13 +1119,10 @@ function getTagListSections(tags: Tag[], recentlyUsedTags: string[], selectedOpt
     const filteredTags = enabledTags.filter((tag) => !selectedOptionNames.includes(tag.name));
 
     if (selectedOptions.length) {
-        const selectedTagOptions = selectedOptions.map((option) => {
-            const tagObject = tags.find((tag) => tag.name === option.name);
-            return {
-                name: option.name,
-                enabled: !!tagObject?.enabled,
-            };
-        });
+        const selectedTagOptions = selectedOptions.map((option) => ({
+            name: option.name,
+            enabled: true,
```

Comment:
Sorry if I'm missing something obvious, but why this change?

Reply:
This PR will make the selected tag display even if it is disabled. So we also need to allow the user to unselect the tag in this case by passing `enabled: true`

Reply:
That makes sense. Please add a comment above this line explaining that